README.md

```diff
@@ -79,6 +79,247 @@ For `testStart` and `testEnd`, the corresponding test object is passed to the re
 
 When `runStart`, `suiteStart` and `testStart` are emitted, the `status`, `runtime` and `errors` attributes are `undefined`.
 
+## Example
+
+Lets take the below testsuite and see how the reporting shoud be, based on the standard presented above. For implementations examples please check [Usage](#usage) and [Integrations](#integrations) below for implementations in real tools.
```

Comment:
s/shoud/should

s/implementations/implementation (the plural s is already on "examples")

Let's drop "for implementations in real tools", that's redundant.